src/blueapi/cli/cli.py

```diff
@@ -191,10 +191,21 @@ def store_finished_event(event: WorkerEvent) -> None:
             finished_event.append(event)
 
     parameters = parameters or "{}"
-    task = Task(name=name, params=json.loads(parameters))
-
-    resp = client.create_task(task)
-    task_id = resp.task_id
+    task_id = ""
+    parsed_params = json.loads(parameters) if isinstance(parameters, str) else {}
+    try:
+        task = Task(name=name, params=parsed_params)
+        resp = client.create_task(task)
+        task_id = resp.task_id
+    except ValidationError:
+        pprint(f"failed to validate the task parameters, {task_id}")
```

Comment:
Can you also print/raise the validation error so the user gets details of what parameters they got wrong?

Reply:
yeah, sure